native/cocos/engine/Engine.cpp

```diff
@@ -140,6 +140,7 @@ int32_t Engine::init() {
 
 void Engine::destroy() {
     cc::DeferredReleasePool::clear();
+    cc::network::HttpClient::destroyInstance();
```

Comment:
Should add macro to use network related codes.

Reply:
currently, we don't have network related macro. Only have USE_SOCKET/ USE_WEBSOCKET_SERVER. But it seems not applicable in this case

Reply:
Ok.